src/R/send-charts.R

```diff
@@ -0,0 +1,203 @@
+library(dplyr)
+library(ggplot2)
+library(reshape2)
+library(stringr)
+
+### Variables for all charts ###
+
+df_historical <- read.csv("../../target/historic-data.csv")
```

Comment:
It does, I've been looking at this and actually because this is being run with RScript I wonder if RProject file may be redundant here? Anyone Rscript takes the path of the script as the root, so using `../../` is always going to work provided everything else in the model does